### ISSUE TITLE
Опционализация

### ISSUE THREAD

**Issue by Skorik99** [iyphc/ecg-cnn-comparator#6]
Цель большинства проектов -- проверять рабочие гипотезы. Гипотезы могут быть разнообразными и варьироваться в зависимости от результатов предыдущих гипотез. Это требует гибкости кода по части выбора различных параметров:
1. Какую модель использовать? base_model.py или resnet.py?;
2. Если я выбрал модель base_model.py -- использую ли я внутри модели batch_norm / dropout?;
3. Если я выбрал resnet.py -- какое значение в dropout мне использовать?

Т.е. опционализация может быть многоуровневой (модель --> параметры в выбранной модели). 
В этом смысле, простые утилиты типа [argparse](https://docs.python.org/3/library/argparse.html) не подходят.
Для конфигурирования проекта предлагается познакомиться с [гидрой](https://hydra.cc/docs/intro/) и с помощью неё опционализировать основные компоненты обучения: оптимизатор, модель, параметры обучения типа batch_size, learning rate итп.

**Comment by Skorik99** [comment]
Также в этой issue можно пофиксить структуру проекта: сейчас в readme есть папка configs, которой нет в проекте. Файлы типа `train.py`/ `predict.py` явно не относятся к папке models. [Сделать папки пакетами](https://packaging.python.org/en/latest/tutorials/packaging-projects/)

**Comment by iyphc** [comment]
[Ссылка на pull request](https://github.com/iyphc/ecg-cnn-comparator/pull/10)
